src/Workspaces/Core/Portable/Utilities/ObjectPools/Extensions.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace Roslyn.Utilities
+namespace Microsoft.CodeAnalysis
```

Comment:
OK. That's gonna be a bigger change, but I think it's worth it. @CyrusNajmabadi, I wish we had "Move to namespace" refactoring.